spec/requests/regions_spec.rb

```diff
@@ -8,40 +8,146 @@
 # GET /api/regions/:id
 #
 
-describe "Regions API" do
-  it "forbids access to regions without an appropriate role" do
-    api_basic_authorize
+RSpec.describe "Regions API", :regions do
+  context "authorization", :authorization do
+    it "forbids access to regions without an appropriate role" do
+      api_basic_authorize
 
-    get(api_regions_url)
+      get(api_regions_url)
 
-    expect(response).to have_http_status(:forbidden)
+      expect(response).to have_http_status(:forbidden)
+    end
+
+    it "forbids access to a region resource without an appropriate role" do
+      api_basic_authorize
+
+      region = FactoryBot.create(:miq_region, :region => "2")
+
+      get(api_region_url(nil, region))
+
+      expect(response).to have_http_status(:forbidden)
+    end
   end
 
-  it "forbids access to a region resource without an appropriate role" do
-    api_basic_authorize
+  context "get", :get do
+    it "allows GETs of a region" do
+      api_basic_authorize action_identifier(:regions, :read, :resource_actions, :get)
+
+      region = FactoryBot.create(:miq_region, :region => "2")
+
+      get(api_region_url(nil, region))
+
+      expect(response).to have_http_status(:ok)
+      expect(response.parsed_body).to include(
```

Comment:
what about `expect_single_resource_query` instead of the 2 expects?

I'd advocate extensive use of the helpers that we have in the project. WDYT?

Reply:
Done!